nservicebus/azure-storage-queues/performance-tuning.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Azure Storage Transport Performance Tuning
+summary: Performance Tuning Azure Storage as transport
+reviewed: 2016-04-21
+tags:
+- Azure
+- Cloud
+- Persistence
```

Comment:
Why do we have both Persistence and Transport tags? There are separate articles.